src/codeu/chat/server/Server.java

```diff
@@ -112,6 +116,15 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
 
         final String name = Serializers.STRING.read(in);
         final User user = controller.newUser(name);
+        
+        String userAddCommand = "U-ADD ";
+        userAddCommand += user.id + " ";
+        userAddCommand += user.name + " ";
+        userAddCommand += user.creation;
+
+        persistentLog.add(userAddCommand);
+        System.out.println(userAddCommand);
+        

```

Comment:
^ I just looked and I think I took this line of code out in my final commit though :)